The build is failing with:
```
2022-01-27T20:24:38.2826031Z AsmHelpers.asm.obj : error LNK2019: unresolved external symbol JIT_WriteBarrier_Table_Loc referenced in function LazyMachStateCaptureState
2022-01-27T20:24:38.2835441Z   Hint on symbols that are defined and could potentially match:
2022-01-27T20:24:38.2853622Z     "void * JIT_WriteBarrier_Table_Loc" (?JIT_WriteBarrier_Table_Loc@@3PEAXEA)
```

--- a/src/coreclr/vm/threads.cpp
+++ b/src/coreclr/vm/threads.cpp
@@ -1125,10 +1125,10 @@ extern "C" void *JIT_WriteBarrier_Loc;
 #ifdef TARGET_ARM64
 extern "C" void (*JIT_WriteBarrier_Table)();
 
-extern void *JIT_WriteBarrier_Loc;
+extern "C" void *JIT_WriteBarrier_Loc;
 void *JIT_WriteBarrier_Loc = 0;
 
-extern void *JIT_WriteBarrier_Table_Loc;
+extern "C" void *JIT_WriteBarrier_Table_Loc;
 void *JIT_WriteBarrier_Table_Loc = 0;
 #endif // TARGET_ARM64
 

